File: GCD74.java
import java.util.Scanner;

public class HW74_GCDjava {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter first value: ");
        int x = input.nextInt();
        System.out.print("Enter second value: ");
        int y = input.nextInt();
        int gcd = 1;
        for (int i = 1; i <= x && i <= y; i++) {
            if (x % i == 0 && y % i == 0) {
                gcd = i;


            }



        }
        System.out.println(gcd);

    }
}

File: GCD74.py
x = int(input("enter first value: "))
y = int(input("enter second value: "))

if x < y:
        b = x
else:
        b = y
for n in range(1, b + 1):
    if (x % n == 0)and(y % n == 0):
        GDC = n


print("GCD is", GDC)